src/terraform/templates/aws.tf

```diff
@@ -26,7 +26,7 @@ variable "key_name" {
 }
 
 module "aws" {
-  source = "git@github.com:Originate/exosphere.git//terraform//aws?ref={{terraformCommitHash}}"
+  source = "github.com/Originate/exosphere.git//terraform//aws?ref={{terraformCommitHash}}"
```

Comment:
This switches to downloading via https so we don't need to mount a user's github credentials (and so they don't need a github account at all).